tests/model/model_import_export.test.ts

```diff
@@ -486,7 +486,8 @@ describe("Import", () => {
     expect(Object.keys(getMerges(model))).toHaveLength(0);
     activateSheet(model, sheet1);
     expect(Object.keys(getMerges(model))).toHaveLength(1);
-    expect(Object.values(getMerges(model))[0].topLeft).toEqual(toCartesian("A2"));
+    const mergeZone = getMerges(model)[1];
+    expect({ col: mergeZone.left, row: mergeZone.top }).toEqual(toCartesian("A2"));
```

Comment:
Check the entire merge and not only the first one
```suggestion
    expect(getMerges(model)).toEqual([toZone("A2:B2")]);
```

Reply:
@LucasLefevre it's not a list, it's an object with the ids of merges as keys
![image](https://github.com/odoo/o-spreadsheet/assets/61025089/11cff7b2-65d0-4d33-afe6-37b396de060f)

Reply:
Ah ok,
then you can still check the entire merge
```ts
expect(mergeZone).toEqual(toZone("A2:B2"));
```